backend/utils/authUtils.js

```diff
@@ -40,7 +51,7 @@ exports.signUserUp = (email, password, username) => {
         cognito.signUp(cognitoParams, function (err, res) {
             if (err)
                 reject(err)    // Example error codes: InvalidPasswordException, InvalidParameterException (invalid email address)
-            resolve(res)
+            resolve(res.UserSub)
```

Comment:
Codacy has a fix for the issue: [Replace `······resolve(res.UserSub)` with `resolve(res.UserSub);`](https://app.codacy.com/gh/cpen321project/cpen321project/pullRequest?prid=9852555)
```suggestion
      resolve(res.UserSub);
```